ansible/tasks/postgres-extensions/19-pg_graphql.yml

```diff
@@ -1,69 +1,14 @@
 # pg_graphql
-- name: pg_graphql - download & install dependencies
-  apt:
-    pkg:
-      - make
-      - gcc
-      - pkg-config
-      - clang
-      - libssl-dev
-    update_cache: yes
-    install_recommends: no
-
-- name: download rust/cargo installer
-  become: yes
-  get_url:
-    url: https://sh.rustup.rs
-    dest: /tmp/sh.rustup.rs
-    mode: '0755'
-    owner: postgres
-    group: postgres
-
-- name: install rust/cargo
-  become: yes
-  become_user: postgres
-  shell: /tmp/sh.rustup.rs -y
-
-- name: install cargo-pgx
-  become: yes
-  become_user: postgres
-  shell:
-    cmd: "~/.cargo/bin/cargo install --version '={{ pg_graphql_pgx_version }}' cargo-pgx && ~/.cargo/bin/cargo pgx init --pg{{ postgresql_major }} pg_config"
-
-- name: pg_graphql - download latest release
-  become: yes
-  git:
-    repo: https://github.com/supabase/pg_graphql.git
-    dest: /tmp/pg_graphql
-    version: "{{ pg_graphql_release }}"
-
-- name: pg_graphql - temporarily transfer ownership to postgres
-  become: yes
-  file:
-    path: '{{ item }}'
-    recurse: yes
-    owner: postgres
-    group: postgres
-  with_items:
-    - /tmp/pg_graphql
-    - /usr/lib/postgresql/lib # AMI
-    - /usr/lib/postgresql/14/lib # Docker Image
-    - /usr/share/postgresql/14
-
-- name: pg_graphql - install
-  become: yes
-  become_user: postgres
-  shell:
-    cmd: "~/.cargo/bin/cargo pgx install --release"
-    chdir: /tmp/pg_graphql
-
-- name: pg_graphql - return ownership to root
-  file:
-    path: '{{ item }}'
-    recurse: yes
-    owner: root
-    group: root
-  with_items:
-    - /usr/lib/postgresql/lib
-    - /usr/lib/postgresql/14/lib
-    - /usr/share/postgresql/14
+- name: Setting arch (x86)
+  set_fact:
+    arch: "amd64"
+  when: platform == "amd64"
+
+- name: Setting arch (arm)
+  set_fact:
+    arch: "arm64"
```

Comment:
you can just use `{{ platform }}` directly; the separate fact is only needed when e.g. the artifact url uses x86 or aarch instead